cdc/model/owner.go

```diff
@@ -269,8 +269,8 @@ func (p ProcessorsInfos) String() string {
 }
 
 // ChangeFeedStatus stores information about a ChangeFeed
+// It is stored in etcd.
 type ChangeFeedStatus struct {
```

Comment:
Please add some comments to explain that `AdminJobType ` should be removed in future.